README.rst

```diff
@@ -339,7 +339,7 @@ Choose from the list of available rules:
     structures; defaults to ``'same'``
   - ``position_after_functions_and_oop_constructs`` (``'next'``, ``'same'``): whether
     the opening brace should be placed on "next" or "same" line after
-    classy constructs (non-anonymous classes, interfaces, traits, methods
+    class constructs (non-anonymous classes, interfaces, traits, methods
```

Comment:
`class constructs (non-anonymous classes, interfaces, traits, ...)`

interface is not `class`, it's class-alike, thus `classy`